setup.py

```diff
@@ -64,7 +77,7 @@
         "src/pystack/_pystack/unwinder.cpp",
         "src/pystack/_pystack/version.cpp",
     ],
-    libraries=["elf", "dw"],
+    libraries=library_flags['libraries'],
```

Comment:
Doing it this way won't include any library directories or include directories reported by `pkg-config`. Instead, we should remove the `libraries=` entirely, and add `**library_flags` as the last argument to the `setup` call.